tests/e2e/CHANGELOG.md

```diff
@@ -3,6 +3,83 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 5.0.0 (2022-12-20)
```

Comment:
@Haroenv Does it seem correct to you?

Reply:
no, I think what happened is that because these are local packages that aren't published, lerna doesn't know about the 4.0.0 tag (we should add it in shipjs) and thus always does a from-scratch diff.